config/default/core.entity_view_display.paragraph.hs_text_area.default.yml

```diff
@@ -18,7 +21,12 @@ content:
     weight: 0
     label: hidden
     settings: {  }
-    third_party_settings: {  }
+    third_party_settings:
+      field_formatter_class:
+        class: 'color-[paragraph:field_hs_text_area_bg_color]'
```

Comment:
These are not the most semantic names but I see why you did this. I think for H&S this is fine but overall, these should have human-readable labels.